### ISSUE TITLE
fail to run test 'test_container_probing'

### ISSUE THREAD

**Issue by hubutui** [mikeboers/PyAV#543]
I'm trying to create a package for ArchLinux. When runs the test, I got: 
```text
======================================================================
FAIL: test_container_probing (tests.test_file_probing.TestDataProbe)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/python-av/src/PyAV-6.2.0/tests/test_file_probing.py", line 91, in test_container_probing
    'uid': '4482d537-4203-ea40-9e4e-08a22900dd39',
AssertionError: {'operational_pattern_ul': '060e2b34.040101[674 chars]ent'} != {'application_platform': 'AAFSDK (MacOS X)'[609 chars]d39'}
Diff is 764 characters long. Set self.maxDiff to None to see it.

----------------------------------------------------------------------
Ran 157 tests in 122.163s

FAILED (failures=1, skipped=3)
Test failed: <unittest.runner.TextTestResult run=157 errors=0 failures=1>
error: Test failed: <unittest.runner.TextTestResult run=157 errors=0 failures=1>
```

**Comment by jlaine** [comment]
Could you post the full diff as it is truncated ?

**Comment by hubutui** [comment]
I build it with this [PKGBUILD](https://github.com/archlinuxcn/repo/blob/master/archlinuxcn/python-av/PKGBUILD). To reproduce, you need:
1. ArchLinux environment with `devtools` installed
2. Download this `PKGBUILD`
3. Build this package in a clean chroot: `extra-x86_64-build`

[log.txt](https://github.com/mikeboers/PyAV/files/3512642/log.txt)

**Comment by jlaine** [comment]
Unfortunately the diff is still truncated, you need to modify the test_file_probing.py file and add `maxDiff = None` like this:

```python
class TestDataProbe(TestCase):
    maxDiff = None

    def setUp(self):
        self.file = av.open(fate_suite('mxf/track_01_v02.mxf'))
```

**Comment by hubutui** [comment]
after `test_file_probing.py`:
```python
class TestDataProbe(TestCase):
    def setUp(self):
        self.maxDiff = None
        self.file = av.open(fate_suite('mxf/track_01_v02.mxf'))
```
here is the [log.txt](https://github.com/mikeboers/PyAV/files/3568307/log.txt)

**Comment by jlaine** [comment]
I believe this was fixed by 76e090dcb8ecb54d1c0c520774a885e8f2cf734c
